tests/lit-tests/2280.ispc

```diff
@@ -0,0 +1,33 @@
+// RUN:     %{ispc} %s --target=host --nostdlib --emit-llvm-text -E                       -o -
+// RUN:     %{ispc} %s --target=host --nostdlib --emit-llvm-text                          -o - | FileCheck %s -check-prefix=CHECK_PASS_0
+// RUN:     %{ispc} %s --target=host --nostdlib --emit-llvm-text                          -o - | FileCheck %s -check-prefix=CHECK_PASS_2
```

Comment:
Is the only way to merge these into two checks using the same output to use a file? `%-NEXT` has to match the following line, and `%-SAME` has to match the same line.

Reply:
No, you just need to have to lines with corresponding prefix, that's it. I.e.:
```
// CHECK_PASS_0: ret i32 1337
// CHECK_PASS_0: ret i32 7331
```